modules/admin/proc-rbac-config-conditional-policy-file.adoc

```diff
@@ -0,0 +1,118 @@
+* You have defined conditional policies in a YAML file, which includes role references, permission mappings, and conditions.
++
+--
+The following is an example of a YAML file defining conditional policies:
+
+[source,yaml]
+----
+---
+result: CONDITIONAL
+roleEntityRef: 'role:default/test'
+pluginId: catalog
+resourceType: catalog-entity
+permissionMapping:
+  - read
+  - update
+conditions:
+  rule: IS_ENTITY_OWNER
+  resourceType: catalog-entity
+  params:
+    claims:
+      - 'group:default/team-a'
+      - 'group:default/team-b'
+---
+result: CONDITIONAL
+roleEntityRef: 'role:default/test'
+pluginId: catalog
+resourceType: catalog-entity
+permissionMapping:
+  - delete
+conditions:
+  rule: IS_ENTITY_OWNER
+  resourceType: catalog-entity
+  params:
+    claims:
+      - 'group:default/team-a'
+----
+--
+* You have defined roles and associated policies in a CSV file that serves as a basis for creating roles and permissions.
++
+For more information, see xref:ref-rbac-conditional-policy-definition_title-authorization[].
+
+.Procedure
+
+. In {ocp-short}, open `app-config.yaml` file and specify the path to both the `conditionalPoliciesFile` and the `policies-csv-file` as shown in the following example:
++
+--
+.Example `app-config.yaml` file
+[source,yaml]
+----
+permission:
+  enabled: true
+  rbac:
+    conditionalPoliciesFile: /some/path/conditional-policies.yaml
+    policies-csv-file: /some/path/rbac-policy.csv
+----
+--
+
+. To enable automatic reloading of the policy file without restarting the application, add the `policyFileReload` option and set it to `true`:
++
+--
+.Example `app-config.yaml` file 
+[source,yaml]
+----
+permission:
+  enabled: true
+  rbac:
+    conditionalPoliciesFile: /some/path/conditional-policies.yaml
+    policies-csv-file: /some/path/rbac-policy.csv
+    policyFileReload: true
+----
+--
+
+. Optional: Define nested conditional policies in the YAML file as needed.
++
+--
+.Example for nested conditional policies
+[source,json]
+----
+{
+  "result": "CONDITIONAL",
+  "roleEntityRef": "role:default/developer",
+  "pluginId": "catalog",
+  "resourceType": "catalog-entity",
+  "permissionMapping": ["delete"],
+  "conditions": {
+    "anyOf": [
+      {
+        "rule": "IS_ENTITY_OWNER",
+        "resourceType": "catalog-entity",
+        "params": {"claims": ["$ownerRef"]}
+      },
+      {
+        "rule": "IS_ENTITY_KIND",
+        "resourceType": "catalog-entity",
+        "params": {"kinds": ["Group"]}
+      },
+      {
+        "not": {
+          "rule": "IS_ENTITY_KIND",
+          "resourceType": "catalog-entity",
+          "params": { "kinds": ["Api"] }
+        }
+      }
+    ],
+  }
+}
+----
+
+In the previous example, the `role:default/developer` is granted the condition to delete catalog entities only if they are the entity owner or if the catalog entity belongs to a group. However, this condition does not apply if the catalog entity is an API.
```

Comment:
I think you provided an excellent example! However, the condition doesn’t quite work as described. To make it work properly, we should:

```json
{
  "allOf": [
    {
      "anyOf": [
        {
          "rule": "IS_ENTITY_KIND",
          "resourceType": "catalog-entity",
          "params": {
            "kinds": [
              "group"
            ]
          }
        },
        {
          "rule": "IS_ENTITY_OWNER",
          "resourceType": "catalog-entity",
          "params": {
            "claims": [
              "$ownerRefs"
            ]
          }
        }
      ]
    },
    {
      "not": {
        "rule": "IS_ENTITY_KIND",
        "resourceType": "catalog-entity",
        "params": {
          "kinds": [
            "api"
          ]
        }
      }
    }
  ]
}
```

Reply:
Added to doc